src/node_report.cc

```diff
@@ -680,6 +680,7 @@ static void PrintResourceUsage(JSONWriter* writer) {
     writer->json_objectend();
   }
   writer->json_objectend();
+  // Maybe we can remove this
 #ifdef RUSAGE_THREAD
```

Comment:
`uv_getrusage_thread` internally switches by `RUSAGE_THREAD`, any reason to keep this branch?

https://github.com/libuv/libuv/blob/3a9a6e3e6bc78565ddf94cf462c9877c1004bb62/src/unix/core.c#L1106-L1107

Reply:
Yes, for backward compatibility, I'm not sure if they are fully compatible. I can remove it if necessary.

Reply:
Would you mind expanding on the backward compatibility that this duplication can address? Like linked above, when `RUSAGE_THREAD` is defined, the behavior should be equivalent.

Reply:
Make sure the code works for users who previously used macro RUSAGE_THREAD, I am not sure for 'the behavior should be equivalent' although `uv_getrusage_thread` seems compatible.